R/forecast.R

```diff
@@ -292,13 +320,14 @@ validate_forecast.forecast_sample <- function(forecast, forecast_type = NULL, ..
 #' for all forecasts.
 #' @param data A data.table with forecasts and observed values that should
 #' be validated.
+#' @inheritParams assert_forecast
 #' @return returns the input
 #' @importFrom data.table ':=' is.data.table
 #' @importFrom checkmate assert_data_table
 #' @importFrom cli cli_abort cli_inform
 #' @export
 #' @keywords internal_input_check
-validate_general <- function(data) {
+assert_forecast_generic <- function(data, silent = FALSE) {
```

Comment:
its not really silent as it will still flag errors maybe verbose is a better way to put it.